packages/react-components/icons/RadioButton.tsx

```diff
@@ -0,0 +1,30 @@
+import colors from '@celo/react-components/styles/colors'
+import * as React from 'react'
+import Svg, { Circle } from 'svgs'
+
+interface Props {
+  height?: number
+  color?: string
+}
```

Comment:
This won't pass TypeScript build phase as it doesn't support the `enabled` property which is used in some places.

Reply:
do you mean the `selected` property of RadioButton? :o